include/fmt/core.h

```diff
@@ -283,6 +283,14 @@ struct monostate {};
 
 namespace detail {
 
+constexpr bool is_constant_evaluated() FMT_DETECTED_NOEXCEPT {
```

Comment:
This should be `FMT_NOEXCEPT`, not `FMT_DETECTED_NOEXCEPT`.

Reply:
Done